content/pages/homepage.html

```diff
@@ -174,6 +174,33 @@ <h3 class="ui large monospace header">
     <section>
       <div class="ui very padded container">
         <div class="ui stackable equal width grid">
+          <div class="stretched row">
+            <div class="column">
+              <div class="ui very padded raised large segment">
+                {% call marketing.testimonial(
+                    name="Max Hilbrunner",
+                    title="Software and game developer",
+                    company="Godot Engine",
+                    image="https://avatars.githubusercontent.com/u/1654763?v=4",
+                    link="https://docs.godotengine.org/") %}
+                  Read the Docs plays a large part in enabling a small volunteer team to maintain our docs. Without the generous Community plan, we couldn't have afforded anything close to the CI/build resources that we got with Read the Docs.
+                {% endcall %}
+              </div>
+            </div>
+            <div class="column">
+              <div class="ui very padded raised large segment">
+                {% call marketing.testimonial(
+                    name="Matt Fisher",
+                    title="Scientific Software Developer",
+                    company="UC Berkeley Eric & Wendy Schmidt Center for Data Science & Environment",
+                    image="https://github.com/mfisher87.png",
+                    link="https://dse.berkeley.edu/") %}
+                  Read the Docs is a critical component of the open source software ecosystem. And their PR builds are the best and most trustworthy implementation out there.
+                {% endcall %}
+              </div>
+            </div>
+          </div>
+          {#
```

Comment:
This just swaps the testimonials for now to get them up, but we'll need to futz with it more if we want more than 2.